### PR TITLE
Added drop method to GridFSBucket

### PR DESCRIPTION
**Pull request by rozza**
JAVA-192

[Review on Reviewable](https://reviewable.io/reviews/rozza/mongo-java-driver/118)

### CODE REVIEW

**Comment by rozza** [comment]
Updated

File: driver/src/main/com/mongodb/client/gridfs/GridFSBucket.java [hunk 1]
```diff
@@ -256,4 +256,9 @@
      */
     void rename(ObjectId id, String newFilename);
 
+    /**
+     * Drops the data associated with this bucket from the Database.
```
Comment (jyemin) [inline driver/src/main/com/mongodb/client/gridfs/GridFSBucket.java]: Sorry, missed this:  lowercase "Database"

**Comment by jyemin** [comment]
lgtm after the two small requests

**Comment by rozza** [comment]
Done rebased and merged.

**Pull request closed without merging**